src/Point.ts

```diff
@@ -0,0 +1,35 @@
+export class Point {
+  constructor(private x: number = 0, private y: number = 0) {
+    this.x = x;
+    this.y = y;
+  }
```

Comment:
Please, follow the task requirements here. 
Instead of using default parameters, we need to have overloaded constructor.
```
default constructor which creates a point at the location of (0, 0);
overloaded constructor (use multiple constructors declaration for Typescript) which creates a point by x and y coordinates;
```

Reply:
Constructor & Method override applied